### DISCUSSION TITLE
plasma-manager: difference between hotkeys and shortcuts

### DISCUSSION THREAD

**Discussion by ZlobnuDima** [nix-community/plasma-manager#372] [Q&A] [answered]
What is the difference between hotkeys and shortcuts in the KDE context? When should I use one or the other?

**Comment by HeitorAugustoLN** [comment, marked as answer]
Shortcuts module should be used for setting KDE Plasma shortcuts such as changing current virtual desktop, or launching applications.

Hotkeys should be used to make shortcuts for commands, for example, `dmenu` or `wpctl set-volume -l 1.4 @DEFAULT_AUDIO_SINK@ 5%-`. But if you want to can also be used for launching applications with their respective commands, etc.

You can see examples in my own configuration: https://github.com/HeitorAugustoLN/nix-config/blob/main/home/heitor/features/desktop/plasma/shortcuts.nix

**Reply by ZlobnuDima** [reply]
Thanks.

**Reply by k2on** [reply]
https://github.com/HeitorAugustoLN/nix-config/blob/ee48ae394df4a44b06cc5df34afe0fbce63827df/home/heitor/features/desktop/plasma/shortcuts.nix
